padl/transforms.py

```diff
@@ -165,13 +168,41 @@ def pd_post_load(self, path: Path, i: int):
         :param i: Unique transform index, can be used to construct filenames.
         """
 
+    def pd_zip_save(self, path: Union[Path, str], force_overwrite: bool = False):
+        """Save the transform to a zip-file at *path*.
+
+        The file's name should end with '.padl'. If no extension is given, it will be added
+        automatically.
+
+        If the file exists, call with *force_overwrite* = `True` to overwrite. Otherwise, this
+        will raise a FileExistsError.
+        """
+        path = Path(path)
+        if path.suffix == '':
+            path = path.parent / (path.name + '.padl')
+
+        if path.exists():
+            if not force_overwrite:
+                raise FileExistsError(f'{path} exists, call with *force_overwrite* to overwrite.')
+            try:
+                rmtree(path)
+            except NotADirectoryError:
+                remove(path)
+
+        with TemporaryDirectory('.padl') as dirname:
```

Comment:
We should consider making the `TemporaryDirectory` inside of `path`. People may not have write permissions in other parts of the machine.